proto/validator/accounts/v2/keymanager.proto

```diff
@@ -11,4 +50,35 @@ message SignRequest {
 
     // Raw bytes data the client is requesting to sign.
     bytes data = 2;
```

Comment:
I think we should rename `data` to `signing_root` to be consistent with the spec and avoid confusions.

In spec:
```
 bls.Sign(privkey, signing_root)
```